content/docs/studio/user-guide/model-registry/view-models.md

```diff
@@ -0,0 +1,72 @@
+# View models
+
+You can find all your models in the [models dashboard](#models-dashboard). Each
+model also has a separate [model details page](#model-details-page). Also, all
+models from a given Git repository are also included as
+[`model` columns in the experiment tables](#model-columns-in-the-projects-experiment-table)
+of those projects that connect to this Git repository.
+
+## Models dashboard:
+
+The models in your model registry are organized in a central dashboard that
+facilitates search and discovery.
+
+![](https://static.iterative.ai/img/studio/models-dashboard.png)
+
+You can sort the models in the dashboard by the `Last updated` timestamp.
```

Comment:
This filter won't work anymore - as we're removing the `Last updated` column, and replacing it with the `Last Event` column. I'll add this to the ticket where we track what we need to do.

Added a checklist item in https://github.com/iterative/studio/issues/3941